packages/python/plotly/setup.cfg

```diff
@@ -1,11 +0,0 @@
-[metadata]
-description_file = README.md
-license_files = LICENSE.txt
```

Comment:
Look into whether license files are included in pyproject.toml

Reply:
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-files